manifests/chain.pp

```diff
@@ -1,8 +1,10 @@
 # defined resource which creates all rules for one chain
 # @param policy [Ferm::Policies] Set the default policy for a CHAIN
 # @param chain [Ferm::Chains] name of the chain that should be managed
+# @param disable_conntrack [Boolean] disable/enable usage of conntrack
 define ferm::chain (
   Ferm::Policies $policy,
+  Boolean $disable_conntrack,
```

Comment:
We should provide a default value here as well. I'm not sure if hiera works, but I think it does?